stdlib/public/libexec/swift-backtrace/main.swift

```diff
@@ -511,6 +511,22 @@ Generate a backtrace for the parent process.
     writeln("Backtrace took \(formattedDuration)s")
     writeln("")
 
+    #if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
+    // On Darwin, if Developer Mode is turned off, or we can't tell if it's
+    // on or not, disable interactivity
+    var developerMode: Int32
```

Comment:
Does this work? I thought you had to initialize the value before using `&`, since the compiler doesn't know if it's in or out.

Reply:
You're right, you do. I tested this code off to the side, but of course forgot to copy over that change.